### DISCUSSION TITLE
Match getters (question about fuzzy search)

### DISCUSSION THREAD

**Discussion by oysandvik94** [saghen/blink.cmp#390] [Q&A]
I have a case where my class has a method called "getEtternavn()"

See screenshot: 
![image](https://github.com/user-attachments/assets/a8a8e1af-fae6-4f3f-8a63-cd023e3cedb6)

And with fuzzy search, I would expect to find it by typing just "etternavn"  or "Etternavn", but I don't:
![image](https://github.com/user-attachments/assets/f079a5f9-083a-474a-b99a-7ca66ecb59f1)

Maybe I'm misunderstanding fuzzy search, but it would be nice to match methods like this. 

Some other examples for toString() method:

"tSt" mathes "toString", but "tst" does not, so I assume there is some case sensitivity here.
However "String" does not match "toString"

**Comment by Shougo** [comment]
Hm...  It seems head match is needed.  I don't know it is configurable.
